### PR TITLE
Some Atlassian fixes

### PR DESCRIPTION
**Pull request by ModProg**
I'll collect a few of my fixes here, but they are not complete yet:

## Jira
image
image

### CODE REVIEW

**Comment by ModProg** [comment]
> Nice, these JIRA issues have been annoying me lately.

Yeah, if anyone has some fixes in their config we might be also able to collect them here

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -1563,6 +1563,12 @@ code:first-of-type {
 #jira-issue-header, #jira-issue-header-actions {
     background-color: rgb(29, 32, 33) !important;
 }
+.ak-editor-content-area [colorname="Light green"] {
+  background-color: #073f29 !important;
+}
+.ak-editor-content-area [colorname="Light teal"] {
+  background-color: #003a42 !important;
+}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1571]: Would it be able to use the form of `${...}` where `...` is replaced by the original hex color (see line 1573 for example)? This way dark reader can modify it on-the-fly and according to the user's settings.

Comment (ModProg) [reply src/config/dynamic-theme-fixes.config:1571]: I'll try it next time I'm using it

**Comment by ModProg** [comment]
> Pinging @Gusted.
> 
> I believe this one can be closed. It's been sitting here open for approximately 7 months.

That's why I created it as a draft more as a collection of fixes than something ready to be merged

**Comment by ModProg** [comment]
Actually it turns out they are working on a builtin dark theme, so I'm gonna close this:

https://community.atlassian.com/t5/Jira-articles/Help-us-test-Jira-s-new-dark-theme/ba-p/2211399#M7602

**Pull request closed without merging**